shuffle/cli/src/shared.rs

```diff
@@ -269,6 +267,58 @@ impl Home {
             )),
         }
     }
+
+    pub fn write_network_into_config(&self) -> Result<()> {
+        let network_config_path = self.shuffle_path.join("Networks.toml");
+        let network_config_string = toml::to_string_pretty(&generate_localhost_network()?)?;
+        fs::write(network_config_path, network_config_string)?;
+        Ok(())
+    }
+}
+
+fn generate_localhost_network() -> Result<Value> {
+    let local_host_value = r#"
+        [localhost]
+        base = 'http://127.0.0.1'
+        json_rpc_port = 8080
+        dev_api_port = 8081
+    "#;
+    Ok(toml::from_str::<Value>(local_host_value)?)
+}
+
+pub fn build_url(network_name: &str) -> Result<Url> {
+    let home = Home::new(get_home_path().as_path())?;
+    let network_toml_contents = fs::read_to_string(home.get_networks_config_path())?;
+    let network_toml: Value = toml::from_str(network_toml_contents.as_str())?;
```

Comment:
As mentioned above, there's a `toml::to_string` conterpart to this. We don't want to read the contents as simply `Value` but as a new struct: `NetworkCollection` or some other reasonable name.